draft-ietf-tls-dtls13.md

```diff
@@ -263,19 +263,22 @@ modify their data transmission strategy.
 
 # The DTLS Record Layer
 
-The DTLS record layer is similar to that of TLS 1.3.
-There are three major changes:
+The DTLS record layer is different than the TLS 1.3 record layer and 
+there are changes to previous versions of the DTLS record layer. The
+changes are: 
```

Comment:
changes to previous versions of the DTLS record layer: 

Remove "the changes are"

Reply:
This text is kind of confusing because some of these are TLS -> DTLS changes and some are DTLS 1.2 -> DTLS 1.3 changes. I think I would just remove the bit about the DTLS changes or put at the end "Note that the DTLS 1.3 record layer is different from the DTLS 1.2 record layer."